.github/workflows/upload.yml

```diff
@@ -35,13 +35,24 @@ jobs:
         env:
           CI: false
       - name: Compress
-        run: cd build && tar -czf ../${{env.PACKAGE_NAME}} ./
-      - name: Install upload-assets snap
-        run: sudo snap install upload-assets
+        run: cd build && tar -czf ../${{env.PACKAGE_NAME}} ./ && ls -hs ../${{env.PACKAGE_NAME}}
```

Comment:
Why do we need the `ls -hs` command here?

Reply:
it's for debugging - it prints out the package size